frontend/toolbar/main_toolbar_mobile.html

```diff
@@ -0,0 +1,65 @@
+<md-toolbar md-colors="{background: 'teal-500'}" 
+    layout-align="center" layout="row" ng-if="mainToolbarCtrl.isMobileScreen()" style="overflow: hidden;">
+    <div id="main-toolbar-mobile">
+
+        <md-button class="md-icon-button" aria-label="Menu" 
+            ng-click="mainToolbarCtrl.toggle()" class="main-menu-button">
+            <md-icon>menu</md-icon>
+        </md-button>
+
+        <span ng-if="mainToolbarCtrl.title" class="main-toolbar-title">
+            {{mainToolbarCtrl.title}}
+        </span>
+
+        <div class="extra-menu-container" layout="row" ng-if="mainToolbarCtrl.toolbarMenuItems">
+            <md-menu ng-repeat="item in mainToolbarCtrl.toolbarMenuItems">
+                <md-button ng-click="$mdMenu.open(ev)">
+                    <span style="margin-left: -0.3em;">{{item.title}}</span>
+                    <md-icon style="margin-left: -0.25em;">arrow_drop_down</md-icon>
+                </md-button>
+                <md-menu-content style="width: 80%;">
+                    <md-menu-item ng-repeat="option in item.options">
+                        <md-button ng-click="mainToolbarCtrl.handleOptionAction(item, option)">
+                            <div id="{{option}}">
+                                {{ option }}
+                            </div>
+                        </md-button>
+                    </md-menu-item>
+                </md-menu-content>
+            </md-menu>
+        </div>
+
+        <div ng-if="mainToolbarCtrl.sortByAlpha" ng-click="mainToolbarCtrl.sort()" layout="row" class="sort-button-container">
+            <div>
+                <md-button class="md-icon-button">
+                    <md-icon>sort_by_alpha</md-icon>
+                </md-button>
+            </div>
+        </div>
+
+
+        <div id="search-toolbar-element" ng-if="!mainToolbarCtrl.noSearch" layout="row">
+            <div layout="row" md-theme="input" class="toolbar-search">
+                <md-button class="md-icon-button" title="Buscar" 
+                    ng-click="mainToolbarCtrl.changeState(mainToolbarCtrl.states.SEARCH, {search_keyword: ''})">
```

Comment:
is it necessaty to pass search_keyword as an empty object?

Reply:
yep